src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3587,6 +3728,54 @@ components:
             royalty_config:
               $ref: "#/components/schemas/ComponentRoyaltyConfig"
 
+    StateAccountResourcePreferencesPageRequest:
```

Comment:
nitpick: These two models are inter-changed. Might be a tiny bit nicer to keep the ResourcePreferences and AuthorizedDepositors models next to each other.